src/Signal/Time.purs

```diff
@@ -34,7 +34,8 @@ every = runFn3 everyP constant now
 -- |Returns the number of milliseconds since an arbitrary, but constant, time in the past.
 foreign import now """
   function now() {
-    var perf = typeof performance !== 'undefined' ? performance : null;
+    var perf = typeof performance !== 'undefined' ? performance : null,
+        process = typeof process !== 'undefined' ? process : null;
```

Comment:
That's not going to work. Now `process` will always be `null`. Name it something else.